lib/event-sampler.js

```diff
@@ -0,0 +1,44 @@
+import md5 from './utils/md5';
+
+const projectIdThrottleBlacklist = {
+  // 'PROJECT_ID': THROTTLE_RATE (0.00 - 1.00)
+  xijiKKCqPQzOl0EKa9ta0vmD: 0.10,
+  G6kGCgdy9JzppocYlPmUGpXU: 0.02,
+};
+
+export default class Sampler {
+  constructor(Keen) {
+    this.config = Keen.config;
+    this.projectIdThrottleBlacklist = projectIdThrottleBlacklist;
+  }
+
+  shouldSample(uniqueId) {
+    if (!this.canBeThrottled()) {
+      return true;
+    }
+
+    const id = uniqueId;
```

Comment:
Since the `uniqueId` being passed in here is usually going to be the `session_id` generated by keen AND that `session_id` is just a string of random characters and not *really* a UUID (as initially expected), I don't need to MD5 hash this bad boy and can instead just treat it as a base 36 number (10 digits + 26 AZcharacters).